core/src/test/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIntGroupByFunctionFactoryTest.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.functions.groupby;
+
+import io.questdb.griffin.AbstractGriffinTest;
+import org.junit.Test;
+
+public class CountDistinctIntGroupByFunctionFactoryTest extends AbstractGriffinTest {
```

Comment:
Please add tests where argument is constant and another function for example

```SQL
select count_distinct(l / 3) from long_sequence(100)

select count_distinct(3) from long_sequence(100)
```

Reply:
Added the tests in 1a34b83d527626360eab5826de8b4b56a4a77414